localization/pose_initializer/package.xml

```diff
@@ -19,13 +19,14 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>autoware_cmake</buildtool_depend>
 
+  <depend>autoware_map_height_fitter</depend>
+  <depend>autoware_map_height_fitter</depend>
```

Comment:
duplicated

Reply:
Sorry for my careless mistake. I fixed it at 52d6fa5 .